scripts/test-functional.sh

```diff
@@ -37,16 +38,20 @@ pushd $ML_CORE_TEST_HARNESS_DIR
 
   ## Start the test harness
   echo "==> Starting Docker compose"
-  docker compose --project-name ttk-func --ansi never --profile all-services --profile ttk-provisioning --profile ttk-tests up -d
+  ## For some reason the env vars are not overriding the .env file found in the ttk repo
+  ## So we're setting it in the command
+  ACCOUNT_LOOKUP_SERVICE_VERSION=local docker compose --project-name ttk-func --ansi never --profile all-services --profile ttk-provisioning --profile ttk-tests up -d
 
   echo "==> Running wait-for-container.sh $ML_CORE_TEST_HARNESS_TEST_FUNC_CONT_NAME"
   ## Wait for the test harness to complete, and capture the exit code
-  bash wait-for-container.sh $ML_CORE_TEST_HARNESS_TEST_FUNC_CONT_NAME
-  ## Capture exit code for test harness
-  TTK_FUNC_TEST_EXIT_CODE="$?"
-  echo "==> wait-for-container.sh exited with code: $TTK_FUNC_TEST_EXIT_CODE"
+  TTK_FUNC_TEST_EXIT_CODE=$(docker wait $ML_CORE_TEST_HARNESS_TEST_FUNC_CONT_NAME)
+  echo "==> exited with code: $TTK_FUNC_TEST_EXIT_CODE"
+
+  ## Print docker containers
+  docker ps
```

Comment:
How are you we handling the case when the container doesn't exist at the time of calling docker wait? I think that's the reason we implemented a while loop in the wait-for-container script.

Reply:
The container is created in the up command and it should never be the case that it doesn't exist afaik.

Reply:
ok